src/Randomizer.SMZ3.Tracking/AutoTracking/MetroidStateChecks/SporeSpawn.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Randomizer.SMZ3.Tracking.AutoTracking.MetroidStateChecks
+{
+    /// <summary>
+    /// Metroid state check related to Spore Spawn
+    /// Checks if the player skips Spore Spawn
+    /// </summary>
+    public class SporeSpawn : MetroidStateCheck
+    {
+        /// <summary>
+        /// Executes the check for the current state
+        /// </summary>
+        /// <param name="tracker">The tracker instance</param>
+        /// <param name="currentState">The current state in Zelda</param>
+        /// <param name="prevState">The previous state in Zelda</param>
+        public override bool ExecuteCheck(Tracker tracker, AutoTrackerMetroidState currentState, AutoTrackerMetroidState prevState)
+        {
+            if (currentState.CurrentRegion == 1 && currentState.CurrentRoomInRegion == 22 && prevState.CurrentRoomInRegion == 9)
```

Comment:
Should we add a line in the (extremely unlikely) event that the player enters Spore Spawn the normal way? I have some lines when you _track_ Spore Spawn, but this would be better, I think.

Reply:
I actually have that added to the list. I just have been so against going to Spore Spawn in my tests that I haven't gotten the room number yet. 😂